kuma/static/js/wiki.js

```diff
@@ -191,11 +191,21 @@
         Syntax highlighting scripts
     */
     if($('article pre').length && ('querySelectorAll' in doc)) (function() {
-        var syntaxScript = doc.createElement('script');
-        syntaxScript.setAttribute('data-manual', '');
-        syntaxScript.async = 'true';
-        syntaxScript.src = mdn.staticPath + 'js/syntax-prism-min.js?build=' + mdn.build;
-        doc.body.appendChild(syntaxScript);
+        var bundle = 'syntax-prism';
+        if (window.mdn && mdn.assets.js && mdn.assets.js.hasOwnProperty(bundle)) {
+            mdn.assets.js[bundle].forEach(function(bundle_url, index, array) {
+                var syntaxScript = doc.createElement('script');
+                syntaxScript.setAttribute('data-manual', '');
+                syntaxScript.src = bundle_url;
+                if (index === 0 && array.length > 1) {
+                    /* In development, slow load first script */
```

Comment:
Yes, I'll add more to the comment.  In production, ``syntax-prism-min.js`` contains all five JS files together, evaluated in order.  In development, they are loaded as five separate files.  The later four refer to ``Prism``, which is defined in the first JS file.  We need to slow down an let the first JS file load completely before loading the follow-on files, or get errors about Prism being undefined.